### PR TITLE
WT-13112 Add support for Doxygen 1.11.*

### PR DESCRIPTION
**Pull request by ershov**

### CODE REVIEW

File: src/docs/style/wiredtiger.css [hunk 1]
```diff
@@ -1,3 +1,27 @@
+@media (prefers-color-scheme: dark) {
```
Comment (ajmorton) [inline src/docs/style/wiredtiger.css:1]: Dark mode looks interesting! Do you have a screenshot/link that shows the new output?

**Review by ajmorton — APPROVED** [review]
The changes LGTM and I can confirm there's no changes when I run locally with doxy 1.9.1. 
I'll add @lukech to this review since he's more familiar with doc generation during releases and when we push to develop.

**Comment by lukech** [comment]
As our `doc-update` Evergreen task generates documentation updates for all supported release branches as well, I'd suggest creating a patch build to verify. 

For the few lines of changes in `src/docs/Doxyfile.9`, could we ask people who still use older supported versions (1.8.17, 1.9.1, 1.9.3) in their dev environments to run s_all to make sure no breakage?

File: src/docs/style/wiredtiger.css (hunk 1 above)
Comment (ershov) [reply src/docs/style/wiredtiger.css:1]: I've attached screenshots to the ticket.

**Comment by ershov** [comment]
> As our `doc-update` Evergreen task generates documentation updates for all supported release branches as well, I'd suggest creating a patch build to verify.

Trying to figure it out.

> For the few lines of changes in `src/docs/Doxyfile.9`, could we ask people who still use older supported versions (1.8.17, 1.9.1, 1.9.3) in their dev environments to run s_all to make sure no breakage?

I've tested it on 1.9.1. 1.8 is not affected as it uses another file.

**Comment by ershov** [comment]
[doc-update patch build](https://spruce.mongodb.com/version/666109507f40d70007b8df44) looks good.

**Review by lukech — APPROVED** [review]
LGTM

**Pull request merged**